metaconfig-tests/shared/src/test/scala/metaconfig/ConfDynamicSuite.scala

```diff
@@ -1,4 +1,6 @@
 package metaconfig
+import metaconfig.Configured.NotOk
+import metaconfig.Configured.Ok
```

Comment:
minor, feel free to skip all of this comment:

- i'd add a blank line after package
- i'd import as a wildcard `metaconfig.Configured._`

or i'd simply use `case Configured.Xxx(...)` below, like they did before, without adding imports.

Reply:
done